plugin.yaml

```diff
@@ -796,8 +796,53 @@ node_types:
 
 relationships:
 
-  cloudify.openstack.server_connected_to_keypair:
+  cloudify.relationships.openstack.server_connected_to_keypair:
     derived_from: cloudify.relationships.connected_to
 
-  cloudify.openstack.server_connected_to_server_group:
+  cloudify.relationships.openstack.server_connected_to_server_group:
     derived_from: cloudify.relationships.connected_to
+
+  cloudify.relationships.openstack.server_connected_to_port:
+    derived_from: cloudify.relationships.connected_to
+    target_interfaces:
+      cloudify.interfaces.relationship_lifecycle:
+        postconfigure:
+          implementation: openstacksdk.openstacksdk_plugin.resources.network.port.update
```

Comment:
@EarthmanT We are sure at this moment that the server instance is created right ? I mean the device_id should be ready in order to passed it to the postconfigure action ?